yarn-project/prover-client/src/proving_broker/proving_broker_database/broker_persisted_database.test.ts

```diff
@@ -283,7 +283,7 @@ describe('ProvingBrokerPersistedDatabase', () => {
       batchSize = 5;
 
       config = {
-        dataStoreMapSizeKB: 1024 * 1024 * 1024, // 1GB
+        dataStoreMapSizeKB: 1024 * 1024, // 1GB
```

Comment:
Huh?

Reply:
well it says 1GB in the comment but 1024 * 1024 * 1024kb is 1TB